sig-etcd/charter.md

```diff
@@ -0,0 +1,65 @@
+# SIG etcd Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+[Kubernetes Charter README]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+
+## Scope
+
+Owns the etcd project and how it is used by Kubernetes.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Development of [etcd] and other repositories under [etcd-io organization]
+- Maintenance of [etcd image] packaged with Kubernetes
+
+[etcd]: https://github.com/etcd-io/etcd
+[etcd-io organization]: https://github.com/etcd-io
+[etcd image]: https://github.com/kubernetes/kubernetes/tree/master/cluster/images/etcd
+
+#### Cross-cutting and Externally Facing Processes
+
+- Specifying, testing and improving [The Implicit Kubernetes-ETCD Contract]
+- Release process of etcd and other binaries belonging to [etcd-io organization]
+
+[The Implicit Kubernetes-ETCD Contract]: https://docs.google.com/document/d/1NUZDiJeiIH5vo_FMaTWf0JtrQKCx0kpEaIIuPoj9P6A/edit?usp=sharing
```

Comment:
As this document is referenced in the charter, it shouldn't really be something that isn't version controlled (an edit to the google doc, could mean a possible change in interpretation to the charter). Is it possible to either pull out the things that are relevant to the charter with a reference to the original source, or can we pull this document into something in a git repo that is version controlled?

Reply:
Google doc is versioned but only editor could see the versioning history. Git repo is a good idea. @logicalhan @jpbetz @serathius What do you think? How about we convert this doc to a md doc and put in maybe https://github.com/etcd-io/etcd/tree/main/Documentation/contributor-guide?

Reply:
I would suggest just putting a brief description of what that is here for the purposes of stating "this is a process SIG etcd will own", and if we link out at all, to list it under the subprojects instead.

Reply:
Or we could use https://github.com/etcd-io/etcd/issues/15820? This is versioned and easy to track changed, and is in the subproject. wdyt? @cblecker @BenTheElder

Reply:
> I would suggest just putting a brief description of what that is here for the purposes of stating "this is a process SIG etcd will own", and if we link out at all, to list it under the subprojects instead.

@BenTheElder I have updated the charter md to have a statement saying that sig-etcd owns the implicit kubernetes-etcd contract, instead of a linked doc or github issue, ptal. Thanks!